index.html

```diff
@@ -12,6 +12,28 @@
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
 <body>
+    <header>
+        <nav>
+            <img class="logo-img" src="img/karma-logo.svg" alt="My logo">
+            <ul class="ul-header">
+                <li> <a>login</a> </li>
+                <li> <a>Store</a> </li>
+                <li> <a>Home</a> </li>
+                <li> <a>Blog</a> </li>
+                <li> <a>Help</a> </li>
+            </ul>
+        </nav>
+    </header>
+    <main>
+        <section class="hero-sec">
+            <img class="hero-img" src="img/first-background.jpg" alt="myPic" >
+            <article>Article</article>
```

Comment:
This is a good start! I would encourage you to keep working on this until it looks similar to the example.

Reply:
Thank you. I will try to improve it. Now i am working on "Responsive cake web". I think it has many common things.